src/app/pages/applications/applications.component.scss

```diff
@@ -212,11 +214,22 @@ mat-tab-group {
   }
 
   .ports {
-    bottom: 20px;
-    left: 170px;
-    position: absolute;
+    width: 100%;
+    height: 45px;
+    text-align: left;
+    vertical-align: middle;
+    padding-right: 5px;
+    overflow: hidden;
+	display: flex;
   }
-
+  .ports-text {
+    width: 100%;
+    max-width: max(0px, calc((100% - 50px)*999));
```

Comment:
Can you explain/add about a comment about what this does? I don't see a difference if I remove this line.

Reply:
Will add a comment.

- When there is a slight overlap, the text just get shortned with dots `53/TCP, 53/UD...` 
- However in some cases the buttons don't leave enough room for the ports to show, which ends up with `53`. In those cases (less than 50px width) it removes the text completely. When text is more than 50, it is set to an "endless" max-width, which is limited by the 100% width in practice.

Reply:
Without:

![image](https://user-images.githubusercontent.com/7613738/133754355-77fc5601-294c-4a97-923f-c59e467d4b59.png)


With:

![image](https://user-images.githubusercontent.com/7613738/133754419-491bcb4f-f4b4-4ccb-b014-68398dff1d9f.png)


Simply put:
It complete hides all the text if "just a sliver" would remain, instead of showing just 1, 2 or 3 numbers...